packages/renderer/src/lib/ui/Typeahead.spec.ts

```diff
@@ -196,3 +200,29 @@ test('should navigate in list with keys', async () => {
   // copies the item in the input
   screen.getByDisplayValue('term03');
 });
+
+test('should show clasic border', async () => {
+  render(Typeahead, {
+    initialFocus: true,
+    error: false,
+    delay: 10,
+  });
+
+  const cellOutsideInput = screen.getByRole('cell');
+  expect(cellOutsideInput).not.toHaveClass('border-b-[var(--pd-input-field-stroke-error)]');
+  expect(cellOutsideInput).not.toHaveClass('focus-within:border-[var(--pd-input-field-stroke-error)]');
+  expect(cellOutsideInput).toHaveClass('hover:border-b-[var(--pd-input-field-hover-stroke)]');
```

Comment:
you can check the parent element of the textbox

```suggestion
  const cellOutsideInput = screen.getByRole('textbox');
  const parentInput = cellOutsideInput.parentElement;
  expect(parentInput).not.toHaveClass('border-b-[var(--pd-input-field-stroke-error)]');
  expect(parentInput).not.toHaveClass('focus-within:border-[var(--pd-input-field-stroke-error)]');
  expect(parentInput).toHaveClass('hover:border-b-[var(--pd-input-field-hover-stroke)]');
```